### PR TITLE
CGM Datasets

### PR DESCRIPTION
**Pull request by profxj**
Polished P11

Added in HI Lya from Burchett+16

@jnburchett -- I can add in your CIV too if
you provide that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@ pyigm
 =====
 
 This is a package for the analysis of the Intergalactic Medium
+and the Circumgalactic Medium
 
 Primary Builders include:
 J. Xavier Prochaska
 N. Tejos
+J. Burchett
```
Comment (jnburchett) [inline README.md:10]: Who-hooo!

File: pyigm/cgm/ingest_lit.py [hunk 2]
```diff
@@ -118,18 +122,82 @@ def p11():
...
+
+    for row in b16_vir:
+        # RA, DEC
+        # Galaxy
+        gal = Galaxy((row['ra_gal'], row['dec_gal']), z=row['zgal'])
+        gal.SFR = row['SFR']
+        gal.sig_SFR = row['SFR_err']
+        gal.Mstar = row['mstars']
+        gal.field = row['field']
+        gal.Rvir = row['rrvir'] * u.kpc
```
Comment (jnburchett) [inline pyigm/cgm/ingest_lit.py]: The 'rrvir' column is actually R/Rvir.

File: pyigm/cgm/ingest_lit.py [hunk 3]
```diff
@@ -118,18 +122,82 @@ def p11():
     p11.to_json_tarball(out_file)
 
 
+def ingest_burchett16():
+    """ Ingest Burchett+16
+    """
+    # Virial matching
+    b16_vir_file = resource_filename('pyigm', 'data/CGM/z0/Burchett16_HI_virselect_sfr.fits')
```
Comment (jnburchett) [inline pyigm/cgm/ingest_lit.py]: In testing this, I noticed that the coordinates for the galaxies in the file I have you were incorrect (crazy impact parameters in ion_tbl).  I have revised versions for you.

**Review by jnburchett — CHANGES_REQUESTED** [review]

**Review by jnburchett — COMMENTED** [review]
Awesome, thanks so much!  The only changes necessary that I can see are from new files that I will provide.  I went ahead and prepped the C IV data for you too.

File: pyigm/cgm/ingest_lit.py (hunk 3 above)
Comment (profxj) [reply pyigm/cgm/ingest_lit.py]: ok

File: pyigm/cgm/ingest_lit.py (hunk 2 above)
Comment (profxj) [reply pyigm/cgm/ingest_lit.py]: Dimensionless then?

**Comment by profxj** [comment]
ok, ready for you now @jnburchett 

I have a Notebook on my laptop on home
that I need to add to this PR.  But it is otherwise
good.

**Comment by profxj** [comment]
ok, @jnburchett -- got the CIV in.

please confirm that the EWs for CIV 
refer to the 1548 line.

**Comment by jnburchett** [comment]
Yes, they do!

**Pull request merged**